### ISSUE TITLE
Reduce need consuption by population tier

### ISSUE THREAD

**Issue by suxkar** [NiHoel/Anno1800Calculator#88]
Hello!

First of all, thank you for this amazing tool, I've recently discovered that it is much more complicated and detailed that what I thought it to be, incredibly useful!

Now to the issue: I was trying to "implement" into my island the use of items the nullify cosumption such as "Brasserie Patron Martens" (Pub provides Sausage, Beer, Bread and Church). It will remove, for example, the need for Sausage for Workers, but it will not do so for Artisans, since they do not have the need for Pub (unless I'm wrong and it removes it anyway?). 
"Actor" will have the same problem (it will not remove the lifestyle need for rum of workers because they do not have the need for variety theatre). For this reason, I would need a tier based consumption need tick box, properly implemented item effects or something similar.

I thought of multiple workarounds, but each kinda defeats the purpose of having an island simulator:
1) Uncheking the tick box in the proper population tier menu on top: it would require to manually modify the population/max population numbers and give up automatic updates, which gets very messy if I change my mind while deciding the population numbers.
2) "Subtract" production using dummy traderoute or dockland: again, very messy if I change values, which happens a lot while designing a big island.
3) Calculate needs manually and separately: even more messy

So: am I missing something or what I need cannot be implemented right now?

**Comment by NiHoel** [comment]
> Now to the issue: I was trying to "implement" into my island the use of items the nullify cosumption such as "Brasserie Patron Martens" (Pub provides Sausage, Beer, Bread and Church). It will remove, for example, the need for Sausage for Workers, but it will not do so for Artisans, since they do not have the need for Pub (unless I'm wrong and it removes it anyway?).
> "Actor" will have the same problem (it will not remove the lifestyle need for rum of workers because they do not have the need for variety theatre). For this reason, I would need a tier based consumption need tick box, properly implemented item effects or something similar.

You are right, those items cannot directly be activated but you need a workaround

1: derive residents per need only modifies residents, not max. residents. However, only the latter one are relevant for need calculation. If you want to use skyscraper or quarters, you are forced to disable this setting anyway. 

I wouldn't do 2 + 3 at all.

My recommendation: Disable simple view (this only makes sense if you do not use items) and uncheck the option "Automatically update need consumption based on effects and newspaper" which will become visible. The main page will by extended by production chains for each need and tier. There, you can modify the consumption percentage. Set it to 0%.
